mjcf_to_sdformat/tests/resources/test_mujoco.xml

```diff
@@ -28,5 +28,7 @@
                 dir="0.5 0.2 -0.0" attenuation="0.0 0.01 0.001" diffuse="0.8 0.8 0.8"
                 specular="0.2 0.2 0.2" active="false"/>
       </body>
+      <camera name="fixed" pos="0 -6 2" zaxis="0 -1 0"/>
+      <camera name="target_camera" pos="0 -6 2" zaxis="0 -1 0" target="body2"/>
```

Comment:
Can you add two unnamed cameras here and test that they are assigned unique 
names?